libs/argo-cd/config.jsonnet

```diff
@@ -1,25 +1,20 @@
 local config = import 'jsonnet/config.jsonnet';
+local versions = ['2.4.11', '2.3.7', '2.2.12'];
+local manifests = ['application-crd.yaml', 'appproject-crd.yaml'];
```

Comment:
+1 on this!

Reply:
@davidovich 
[applicationset-crd.yaml](https://github.com/argoproj/argo-cd/blob/master/manifests/crds/applicationset-crd.yaml) does not contain any `description` field that would allow creating at least some sane documentation for this CRD

at first I wanted to do this, but as for me, openAPI Schema in `applicationset` is still raw and I abandoned this idea